benchmarks/check.mk

```diff
@@ -74,33 +79,6 @@ main: dummy
 # example/: dummy
 #	@$(def); run_prm $@ test.prm
 
-annulus/: dummy
-	+@$(def); make_lib $@/plugin
-	@$(def); run_all_prms $@/instantaneous
-	@$(def); run_all_prms $@/transient
-
-blankenbach/: dummy
-	+@$(def); make_lib $@/plugin
-	@$(def); run_all_prms $@
-
-crameri_et_al/:  dummy
-	+@$(def); make_lib $@/case_1
-	@$(def); run_all_prms $@/case_1
-	@$(def); run_all_prms $@/case_2
-
-davies_et_al/: dummy
-	+@$(def); make_lib $@/case-2.3-plugin
-	@$(def); run_prm $@ case-2.1.prm
```

Comment:
Are you sure this works? I vaguely recall that the other prm files resume this file.